docs/pages/docs/guides/choosing-a-database.mdx

```diff
@@ -0,0 +1,51 @@
+import { Markdown } from '../../../components/Markdown';
+import { Well } from '../../../components/primitives/Well';
+import { RelatedContent } from '../../../components/RelatedContent';
+
+# Choosing the right Database
+
+Keystone supports Postgres, MySQL and SQLite database engines. This guide highlights the differences between these engines to help you choose the right one for your project.
+
+!> **Note:** SQLite is not recommended in production except for scenarios like the [embedded Keystone](../walkthroughs/embedded-mode-with-sqlite-nextjs) example
+
+## Case Sensitivity
+
+By default, Postgres is case sensitive, while MySQL and SQLite are not case sensitive. This means when filtering and sorting the results will differ depending on the database type and collation used.
+When using `StringFilter` in a GraphQL query the `mode` defaults to that of the database - case sensitive for Postgres, and collation dependent for MySQL. You can make a query case insensitive by using `mode: insensitive`, there is no way to make a `StringQuery` case sensitive if your database is not case sensitive.
```

Comment:
`mode` only exists on Postgres. Also, `StringFilter` & `StringQuery` -> string filter imo

Reply:
Thanks @mitchellhamilton, I just rewrote this section a bit.